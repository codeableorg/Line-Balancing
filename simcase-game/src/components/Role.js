/** @jsx jsx */

import { jsx } from "@emotion/core";

function Role() {
  const contentStyle = {
    padding: 10,
    color: "#3d3d3d"
  };

  const titleStyle = {
    color: "#3d3d3d",
    fontSize: "1.8em"
  };

  const imageStyle = {
    width: 300,
    margin: "0 auto",
    display: "block"
  };

  const pStyle = {
    color: "#3d3d3d"
  };
  return (
    <main css={contentStyle}>
      <h2 css={titleStyle}>Role</h2>
      <img src="assets/img/rolePic.gif" alt="Role" css={imageStyle} />
      <p css={pStyle}>Role message for students</p>
    </main>
  );
}

export default Role;
