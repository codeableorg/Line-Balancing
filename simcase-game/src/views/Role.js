/** @jsx jsx */
import React from "react";
import { jsx } from "@emotion/core";

function Role() {
  return (
    <main>
      <h2>Role</h2>
      <img
        src="assets/img/rolePic.gif"
        alt="Role"
        css={{ width: 300, display: "block", margin: "0 auto" }}
      />
      <p id="role">Role message for students</p>
    </main>
  );
}

export default Role;
