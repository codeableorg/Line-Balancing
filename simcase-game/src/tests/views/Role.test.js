import React from "react";
import { render } from "@testing-library/react";

import Role from "../../views/Role";

test("render Role", () => {
  const { asFragment } = render(<Role />);
  expect(asFragment()).toMatchSnapshot();
});
