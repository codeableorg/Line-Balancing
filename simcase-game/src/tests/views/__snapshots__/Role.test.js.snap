// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render Role 1`] = `
<DocumentFragment>
  <main>
    <h2>
      Role
    </h2>
    <img
      alt="Role"
      class="css-sw1rh4-Role"
      src="/assets/img/rolePic.gif"
    />
    <p
      id="role"
    >
      Role message for students
    </p>
  </main>
</DocumentFragment>
`;
